app_test_suite/steps/executors/pytest.py

```diff
@@ -65,6 +65,7 @@ def prepare_test_environment(self, exec_info: TestExecInfo) -> None:
         )
         pipenv_env = os.environ
         pipenv_env["PIPENV_IGNORE_VIRTUALENVS"] = "1"
+        pipenv_env.pop("VIRTUAL_ENV")
```

Comment:
Changes look good but just saw CI fails and seems related to this.

```
app_test_suite.errors.ATSTestError: Application deployment failed: 'VIRTUAL_ENV
```

Why is this needed?

Reply:
To avoid passing info about virtualenv when you already are in one. My bad that I used `pop` wrong, this is fixed now.